client/components/button/button.css

```diff
@@ -1,5 +1,75 @@
-.unique-button-class {
-  color: white;
+@import '../typography/typography.css';
+@import '../typography/color.css';
+
+button {
+  padding: 0;
   border: none;
-  background-color: red;
+  font: inherit;
+  color: inherit;
+  background-color: transparent;
+  outline: none;
 }
+
+.button {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 10px 16px;
+  height: 48px;
+  border-radius: 8px;
+}
+
+.primary {
+  background: var(--primary-blue);
+  color: var(--primary-white);
+  transition: background 1s ease;
+}
+
+.primary:disabled {
+  background: var(--button-disabled);
+}
+
+.primary:hover:enabled {
+  background: var(--secondary-blueberry);
+}
+
+.primary:focus-visible:enabled {
+  background: var(--border-focused);
+  color: var(--primary-dark);
+}
+
+.primary:active:enabled {
+  background: var(--secondary-blueberry);
+  border: 3px solid var(--border-focused);
+  color: var(--primary-white);
+}
+
+.ghost {
+  background: transparent;
+  border: 1px solid var(--primary-blue);
+  color: var(--primary-blue);
+  transition: background 1s ease, border 1s ease, color 1s ease;
```

Comment:
I can't see change of border color on ghost button click after adding transition for border in css.
It's because transition is very slow.
Could you think about reducing transition time for border?

Reply:
Yes, I also indicated it later and I have already fixed it.